packages/shared/src/modules/surveys/libs/types/survey-ansver-dto.type.ts

```diff
@@ -0,0 +1,10 @@
+type SurveyAnswerDto = {
+	createdAt: string;
+	id: number;
+	label: string;
+	questionId: number;
+	updatedAt: string;
+	value?: number;
```

Comment:
```suggestion
```
Onboarding answer does not have value. Let's keep onboarding and quiz as separate modules